### PR TITLE
Version 1.3

### PR DESCRIPTION
**Pull request by TureBentzin**
This Version delivers a critical fix for 1.2

Changes:
- MiniMessageInjection (fixed)
- New Replacement System and API
- Better JavaDoc
- Small fixes
- Configurations can now be copied

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,3 +38,4 @@ config/test.properties
 latest.log.lck
 config/messageSystem.properties=
```
Comment (TureBentzin) [inline .gitignore:39]: this file needs a rework with the next update

File: AdventureAPI/src/main/java/net/juligames/core/adventure/api/AdventureAPI.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * Please make sure the modules (AdventureAPI & AdventureCore) are the same version to avoid issues while execution!
  */
 public interface AdventureAPI {
-    String API_VERSION = "1.2";
+    String API_VERSION = "1.3-SNAPSHOT";
```
Comment (Bommels05) [inline AdventureAPI/src/main/java/net/juligames/core/adventure/api/AdventureAPI.java]:
```suggestion
    String API_VERSION = "1.3";
```

File: AdventureCore/src/main/java/net/juligames/core/adventure/AdventureCore.java [hunk 3]
```diff
@@ -14,11 +14,12 @@
  * support it. Because of this the AdventureAPI will only function as intended when bundled with the API you are using!
  */
 public class AdventureCore implements AdventureAPI {
-    final String CORE_VERSION = "1.2";
+    final String CORE_VERSION = "1.3-SNAPSHOT";
```
Comment (Bommels05) [inline AdventureCore/src/main/java/net/juligames/core/adventure/AdventureCore.java]:
```suggestion
    final String CORE_VERSION = "1.3";
```

File: AdventureCore/src/main/java/net/juligames/core/adventure/CoreAdventureTagManager.java [hunk 4]
```diff
@@ -16,17 +17,26 @@
...
 import java.util.List;
+import java.util.Map;
 
 /**
  * @author Ture Bentzin
  * 19.11.2022
  */
 public final class CoreAdventureTagManager implements TagManager, AdventureTagManager {
 
+
```
Comment (Bommels05) [inline AdventureCore/src/main/java/net/juligames/core/adventure/CoreAdventureTagManager.java]:
```suggestion
```

File: Core/src/main/java/net/juligames/core/Core.java [hunk 5]
```diff
@@ -48,10 +48,10 @@ public final class Core implements API {
      * This can be set depending on the build of the Core
      */
     public static final String CORE_BRAND = "Core";
-    public static final String CORE_VERSION_NUMBER = "1.2";
+    public static final String CORE_VERSION_NUMBER = "1.3-SNAPSHOT";
```
Comment (Bommels05) [inline Core/src/main/java/net/juligames/core/Core.java]:
```suggestion
    public static final String CORE_VERSION_NUMBER = "1.3";
```

File: Core/src/main/java/net/juligames/core/Core.java [hunk 6]
```diff
@@ -48,10 +48,10 @@ public final class Core implements API {
...
     public static final String CORE_SPECIFICATION = "Gustav";
-    private static final String BUILD_VERSION = "1.2"; //POM VERSION
-    
+    private static final String BUILD_VERSION = "1.3-SNAPSHOT"; //POM VERSION
```
Comment (Bommels05) [inline Core/src/main/java/net/juligames/core/Core.java]:
```suggestion
    private static final String BUILD_VERSION = "1.3"; //POM VERSION
```

**Review by Bommels05 — CHANGES_REQUESTED** [review]
Good Work! If I ignore the Snapshot errors this PR just brings bug fixes and tiny features!

**Review by Bommels05 — APPROVED** [review]

**Pull request merged**